### ISSUE TITLE
diff output: indicate unchanged component versions

### ISSUE THREAD

**Issue by msymons** [CycloneDX/cyclonedx-cli#71]
I would like the output of the diff `--component-versions` command to (optionally?) include components that have NOT had a version change.

Based on testing with 0.10.2, the output of the diff command contains some really useful information that tells me when a component has been upgraded

```
- jetty-setuid-java@1.0.3
+ jetty-setuid-java@1.0.4

- javax.annotation-api@1.3.1
+ javax.annotation-api@1.3.2

- javassist@3.24.1-GA
+ javassist@3.27.0-GA
```

However,  things are not so clear when I have a BOM with (say) 3 versions of commons-codec and one of the versions is the latest release (so it won't change) and the version clash is resolved in the source project.  This results in a new BOM and diff reports:

```
- commons-codec@1.10
- commons-codec@1.11
```
...which the reader could interpret as implying that commons-codec has been removed (when it has not).  Thus, I suggest that the diff report should include all components,  perhaps using "=" to indicate "unchanged".   Thus, when comparing my BOMs:

```
- commons-codec@1.10
- commons-codec@1.11
= commons-codec@1.15
```

At he start I said "optionally?"   Some users might not be interested in listing unchanged components...  especially with BOMs that contain hundreds (or thousands).

**Comment by coderpatros** [comment]
I'm thinking, that if you are doing a component version diff, the extra context of the unchanged component versions should be default output. Well, in the case that there are multiple versions of the component and one of them has changed.

**Comment by coderpatros** [comment]
Available in v0.11.0
